### PR TITLE
Return class specification

### PR DESCRIPTION
**Pull request by aquaminer**
Result:

### CODE REVIEW

File: src/pocketmine/event/server/DataPacketReceiveEvent.php [hunk 1]
```diff
@@ -39,8 +39,10 @@ public function __construct(Player $player, DataPacket $packet){
 	public function getPacket(){
 		return $this->packet;
 	}
-
+	 /**
```
Comment (TheDiamondYT1) [inline src/pocketmine/event/server/DataPacketReceiveEvent.php]: Terrible formatting for such a simple fix...

**Review by Sandertv — CHANGES_REQUESTED** [review]
I don't really see the point in only adding a PHP doc. Might be better if you'd add Typehints.

**Comment by Muqsit** [comment]
Why a comment and not strict mode?

**Comment by opkiler22789** [comment]
@Muqsit, Sorry, but I do not know these little things

**Comment by madecode15** [comment]
@Muqsit @opkiler22789 How are coding a plugin if you don't know how to type hint?

**Comment by TheDiamondYT1** [comment]
@madecode15 its simple lol. But typehinting is good

**Comment by TheDiamondYT1** [comment]
Im about to make a pr to typehint all events

**Review by SOF3 — CHANGES_REQUESTED** [review]
Considering that network branch is being rewritten, it's not meaningful to add this here right now. Moreover, PhpStorm can detect the field type from the constructor assignment of the field.

**Pull request closed without merging**